pkg/kubeclient/kubelet_client.go

```diff
@@ -366,6 +354,41 @@ func (client *KubeletClient) GetNodeCpuFrequency(node *v1.Node) (float64, error)
 	return nodeFreq, nil
 }
 
+func (client *KubeletClient) GetCpuFrequencyFromJob(node *v1.Node, osArch string) (float64, bool) {
```

Comment:
Nit - let's add some doc to explain the return values?

Reply:
Added.